chia/util/condition_tools.py

```diff
@@ -71,6 +71,45 @@ def agg_sig_additional_data(agg_sig_data: bytes) -> Dict[ConditionOpcode, bytes]
     return ret
 
 
+def make_aggsig_final_message(
+    opcode: ConditionOpcode,
+    msg: bytes,
+    spend: Union[Coin, Spend],
```

Comment:
At some point, we should give `Spend` a `.coin` method so we don't need this weird `Union` type here. But this is fine for now.